doc/policies/authentication.rst

```diff
@@ -304,3 +304,27 @@ dependent on the clients IP address and the user agent.
 .. note:: The AuthCache only works for user authentication, not for
    authentication with serials.
 
+
+.. _policy_challenge_text:
+
+
+challenge_text, challenge_text_header, challenge_test_footer
```

Comment:
Hm, I think we have no docs for the ``challenge_text`` action yet. What do you think about adding a sentence here?

Reply:
Hm.
I do not care.
But will do.

Reply:
done in a94cc884

Reply:
Thanks!